triton/apps/comsol.rst

```diff
@@ -180,5 +180,31 @@ The job is submitted with::
 
   $ sbatch comsol_matlab_livelink.sh
 
+Cluster computing controlled from your windows workstation
+----------------------------------------------------------
+
+The following example shows a working set of settings to `use triton as a remote computation cluster for COMSOL <https://www.comsol.com/blogs/how-to-run-on-clusters-from-the-comsol-desktop-environment/>`_.
+
+Prerequisities:
+
+ * Store ssh-keys in pagent so that you can connect to triton with putty without entering the password.
```

Comment:
I have no idea how to do this, but I guess it shouldn't be explained here.  Maybe we can make a new a brand new "SSH on Windows" page? - it seems important enough to deserve it, all of our current ssh pages focus on command line. But that's for another issue, not here.